Add tutorial about Shoot creation into existing AWS VPC

ialidzhikov opened the pull request:
**What this PR does / why we need it**:
Add tutorial about Shoot creation into existing AWS VPC.

**Which issue(s) this PR fixes**:
Fixes #49

**Special notes for your reviewer**:
/cc @vpnachev 

**Release note**:
<!--  Write your release note:
1. Enter your release note in the below block.
2. If no release note is required, just write "NONE" within the block.

Format of block header: <category> <target_group>
Possible values:
- category:       improvement|noteworthy|action
- target_group:   user|operator
-->
```improvement operator
NONE
```

vpnachev commented on website/documentation/050-tutorials/content/howto/create-shoot-into-existing-aws-vpc/_index.md:
```diff
@@ -0,0 +1,99 @@
+
+### 4. Create the Shoot
+
+Prepare your Shoot manifest (you could check the examples in https://github.com/gardener/gardener/tree/master/example). Put your VPC id to `.spec.cloud.aws.networks.vpc.id`:
```
```suggestion
Prepare your Shoot manifest (you could check the [example](https://github.com/gardener/gardener/tree/master/example) manifests). Put your VPC id in `.spec.cloud.aws.networks.vpc.id`:
```

ialidzhikov commented:
> Could you add a ⚠️ warning that it is not recommended to create shoot cluster into VPC that was created by gardener for other shoot cluster because of infrastructure dependencies?

Nice catch! A note about it is now added.

> I feel that some final words are needed at the end.

Yep, you are right. The final part is now updated.
